models/analytics/intermediate/d_divisions.sql

```diff
@@ -1,15 +1,14 @@
 select
     /* Primary Key */
-    id
-
+    stg_meltano__divisions.id
```

Comment:
Not necessary to use fully qualified field names if there's only one table in the select from clause